README.md

```diff
@@ -28,39 +28,89 @@ $ pip install -U chaostoolkit-instana
 ## Usage
 
 To use the probes and actions from this package, add the following to your
-experiment file:
+experiment file. The from_time and to_time are milliseconds sincee epoche.  If 
+the from_time and to_time are omitted, no events will 
+be returned. If you just specify from_time events will be returned from the time value given to the curent time. 
```

Comment:
In the readme, it's good convention to make variables look different with backticks. For instance \`from_time\`.